cmd/atlas/mux.go

```diff
@@ -17,9 +18,10 @@ type (
 	// Driver implements the Atlas interface.
 	Driver struct {
 		*sql.DB
-		Differ    schema.Differ
-		Execer    schema.Execer
-		Inspector schema.Inspector
+		Differ      schema.Differ
+		Execer      schema.Execer
+		Inspector   schema.Inspector
+		MarshalSpec func(v interface{}, marshaler schemaspec.Marshaler) ([]byte, error)
```

Comment:
1. use `schemaspec.Marshaler`
2. i dont think the driver should deal with the syntax that its caller wants to see. the layer above should deal w/formatting

Reply:
For now, in atlas this part is in the driver so it's attached here.